codegen-core/src/main/kotlin/software/amazon/smithy/rust/codegen/core/smithy/protocols/AwsJson.kt

```diff
@@ -110,11 +112,11 @@ open class AwsJson(
 ) : Protocol {
     private val runtimeConfig = codegenContext.runtimeConfig
     private val errorScope = arrayOf(
-        "Bytes" to RuntimeType.Bytes,
-        "Error" to RuntimeType.GenericError(runtimeConfig),
-        "HeaderMap" to RuntimeType.http.member("HeaderMap"),
-        "JsonError" to CargoDependency.smithyJson(runtimeConfig).asType().member("deserialize::Error"),
-        "Response" to RuntimeType.http.member("Response"),
+        "Bytes" to Bytes.toType().resolve("Bytes"),
```

Comment:
personally, I do actually appreciate the prefix so I know this is a rust library but not a super strong preference